packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md

```diff
@@ -107,39 +132,27 @@ as -> cl: Tokens
  -->
 ![Embedded authentication](/img/embedded-auth-seq-flow.png "Embedded authentication sequence flow diagram")
 
-<EmbeddedBrowserWarning />
+### Embedded summary
 
-## Comparison between deployment models
+* Moderate maintenance may be required. If using a CDN, maintenance is more limited as it is being kept up-to-date by Okta. NPM packages a specific version of the widget, which means that it may need to be updated in the project periodically. Customized or local versions of the Sign-In Widget source code would require regular updating.
```

Comment:
```suggestion
* Moderate maintenance may be required. If using a CDN, maintenance is more limited as it is being kept up-to-date by Okta. NPM packages a specific version of the Widget, which means that it may need to be updated in the project periodically. Customized or local versions of the Sign-In Widget source code require regular updating.
```